README.md

```diff
@@ -0,0 +1,95 @@
+# UrContacts
+> A contact list application
+
+UrContacts is a React contact list app built with full CRUD capabilities. It uses json-server to locally store your data in a JSON file at the backend. Because your data persists on a locally hosted database, your data is completely private.
+
+## Installation and Build
+
+1. Clone GitHub repo into a desired local root directory: 
+    - HTTPS: https://github.com/dyazdani/contact-list.git
+    - SSH: `git@github.com:dyazdani/contact-list.git`
+    - GitHub CLI: `gh repo clone dyazdani/contact-list` 
+
+2. `cd contact-list`
```

Comment:
You don't need to tell them to do this, especially since you don't know exactly how they will download the code. I would recommend removing